requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Generated by /scripts/sync-requirements-txt.sh from Pipfile.lock
```

Comment:
This file looks orphan

Reply:
Thanks for checking! I ran the generation locally in a dirty branch so that's how I caused this.

Reply:
fixed